src/components/accession2/Accession2PlantSiteDetails.tsx

```diff
@@ -0,0 +1,121 @@
+    let min: number | undefined;
+    let max: number | undefined;
+    // we support either a number or a range of two numbers, eg. 2 - 5
+    const numbers = value.split('-').map((num: string) => num.trim());
+    if (numbers.length < 3) {
+      // Parse the numbers
+      min = parseInt(numbers[0], 10);
+      max = parseInt(numbers[1], 10);
+      const noMin = isNaN(min) || min === 0;
+      const noMax = isNaN(max) || max === 0;
+      if (noMin) {
+        min = undefined;
+      }
+      if (noMax) {
+        max = min;
+      } else if (noMin) {
+        // if we have " - 3" use 3 as min and max
+        min = max;
+      } else {
+        // if we have "4 - 3" use 3 as min and 4 as max;
+        // if we have "3 - 4" use 3 as min and 4 as max;
+        const temp = min as number;
+        min = Math.min(temp, max);
+        max = Math.max(temp, max);
+      }
+    }
+    setRecord((prev) => {
+      return {
+        ...prev,
+        plantsCollectedFromMin: min,
+        plantsCollectedFromMax: max,
+      };
+    });
+  };
+
+  if (!isOpen) {
+    return (
+      <Grid item xs={12} marginTop={theme.spacing(2)}>
+        <Box display='flex' justifyContent='flex-start'>
+          <Link sx={{ textDecoration: 'none' }} href='#' id='addPlantSiteDescription' onClick={() => setIsOpen(true)}>
+            {strings.ADD_PLANT_SITE_DESCRIPTION}
+          </Link>
+        </Box>
+      </Grid>
+    );
+  }
+
+  return (
+    <Grid item xs={12} display='flex' flexDirection={'column'} marginTop={theme.spacing(2)}>
+      <Grid item xs={12}>
+        <Select
+          id='collectionSource'
+          label={strings.COLLECTION_SOURCE}
+          placeholder={strings.SELECT}
+          selectedValue={record.collectionSource}
+          options={ACCESSION_2_COLLECTION_SOURCES}
+          onChange={(value) => onChange('collectionSource', value)}
+          fullWidth={true}
+          readonly={true}
+        />
+      </Grid>
+      <Grid item xs={12} display='flex' flexDirection={isMobile ? 'column' : 'row'} marginTop={theme.spacing(2)}>
+        <Grid item xs={gridSize()} sx={{ marginRight: isMobile ? 0 : theme.spacing(2) }}>
+          <Textfield
+            id='numPlants'
+            value={numPlants}
+            onChange={(unused, value) => onChangeNumPlants(value)}
+            type='text'
+            label={strings.NUMBER_PLANTS_COLLECTED_FROM}
+          />
+        </Grid>
+        <Grid item xs={gridSize()} sx={{ marginTop: isMobile ? theme.spacing(2) : 0 }}>
+          <Textfield
+            id='founderId'
+            value={record.founderId}
+            onChange={onChange}
+            type='text'
+            label={strings.PLANT_ID + ' ' + strings.IF_APPLICABLE}
+            tooltipText={strings.PLANT_ID_TOOLTIP}
+          />
+        </Grid>
+      </Grid>
+      <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
+        <Textfield
+          id='collectionSiteNotes'
+          value={record.collectionSiteNotes}
```

Comment:
This property will change after BE support. For now I'd like to get it in this way. I'll follow up once BE work is done.

Reply:
I am working on the BE task, I will just update this review after BE work is finished. Sorry about the confusion.